packages/synapse-react-client/src/components/EntityHeaderTable/EntityHeaderTable.tsx

```diff
@@ -255,21 +282,7 @@ export const EntityHeaderTable = (props: EntityHeaderTableProps) => {
     })
     setRefsInState(newRowRefs)
   }
-  const selectableTypes = [
-    EntityType.PROJECT,
-    EntityType.FOLDER,
-    EntityType.FILE,
-    EntityType.TABLE,
-    EntityType.LINK,
-    EntityType.ENTITY_VIEW,
-    EntityType.DOCKER_REPO,
-    EntityType.SUBMISSION_VIEW,
-    EntityType.DATASET,
-    EntityType.DATASET_COLLECTION,
-    EntityType.MATERIALIZED_VIEW,
-    EntityType.VIRTUAL_TABLE,
-  ]
-
+  ;``
```

Comment:
```suggestion

```

Reply:
Took me a second, but I agree with the suggested change of removing this :D

Reply:
I'm surprised ESLint didn't have a problem with this!